### ISSUE TITLE
sketch issue

### ISSUE THREAD

**Issue by jpgagnon73** [mattrcampbell/EOSEncoderWing#3]
Matt, I now have a teensy 3.6, which is what you used.  I am getting a "wire not defined in scope" error (see pic).  Can't seem to figure this out.

Also, on the teensy 3.6, which pins did you use for the displays.  I'm not familiar with the OLED displays and don't see in the sketch which pins they would attach to.

I've finally printed my enclosure for this and am excited to get it all wired up.

Thanks
<img width="1071" alt="encoder wing" src="https://user-images.githubusercontent.com/23482414/33916701-16d0cc56-df78-11e7-953a-babded094f83.png">

JP

**Comment by mattrcampbell** [comment]
Where did you install the modified Adafruit library?  I'm guessing that it is finding the standard version first and using that one. IIRC Wire is defined in i2c_t3.h which is included in the modified version I made.

**Comment by jpgagnon73** [comment]
I still don't get it.  I created a new user on my Mac and re-installed everything.  still have issues.  see my error logs.

Arduino: 1.8.5 (Mac OS X), TD: 1.40, Board: "Teensy 3.6, Serial, 180 MHz, Faster, US English"

EOSEncoderWing: In function 'void setup()':
EOSEncoderWing:135: warning: ISO C++ forbids converting a string constant to 'char*' 
     printMessage(3, "SD FAIL");
                              ^
EOSEncoderWing:137: warning: ISO C++ forbids converting a string constant to 'char*' 
     printMessage(3, "SD INIT");
                              ^
EOSEncoderWing: In function 'void chanMessage(OSCMessage&, int)':
EOSEncoderWing:245: warning: ISO C++ forbids converting a string constant to 'char*' 
         setWheelName(i, "Intens", 1);
                                    ^
EOSEncoderWing: In function 'int parseOSCMessage(char*, int)':
EOSEncoderWing:260: warning: ISO C++ forbids converting a string constant to 'char*' 
     printMessage(0, "Handshake");
                                ^
/Users/arduino/Documents/Arduino/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp: In member function 'virtual void Adafruit_SSD1306::drawPixel(int16_t, int16_t, uint16_t)':
/Users/arduino/Documents/Arduino/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp:38:14: error: 'swap' was not declared in this scope
     swap(x, y);
              ^
/Users/arduino/Documents/Arduino/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp:38:14: note: suggested alternatives:
In file included from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/tools/arm/arm-none-eabi/include/c++/5.4.1/utility:70:0,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/tools/arm/arm-none-eabi/include/c++/5.4.1/algorithm:60,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/teensy/avr/cores/teensy3/wiring.h:67,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/teensy/avr/cores/teensy3/WProgram.h:45,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/teensy/avr/cores/teensy3/Arduino.h:3,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/teensy/avr/libraries/Adafruit_GFX/Adafruit_GFX.h:5,
                 from /Users/arduino/Documents/Arduino/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp:24:
/Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/tools/arm/arm-none-eabi/include/c++/5.4.1/bits/stl_pair.h:254:5: note:   'std::swap'
     swap(pair<_T1, _T2>& __x, pair<_T1, _T2>& __y)
     ^
In file included from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/tools/arm/arm-none-eabi/include/c++/5.4.1/exception:162:0,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/tools/arm/arm-none-eabi/include/c++/5.4.1/new:40,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/tools/arm/arm-none-eabi/include/c++/5.4.1/bits/stl_construct.h:59,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/tools/arm/arm-none-eabi/include/c++/5.4.1/bits/stl_tempbuf.h:60,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/tools/arm/arm-none-eabi/include/c++/5.4.1/bits/stl_algo.h:62,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/tools/arm/arm-none-eabi/include/c++/5.4.1/algorithm:62,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/teensy/avr/cores/teensy3/wiring.h:67,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/teensy/avr/cores/teensy3/WProgram.h:45,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/teensy/avr/cores/teensy3/Arduino.h:3,
                 from /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/teensy/avr/libraries/Adafruit_GFX/Adafruit_GFX.h:5,
                 from /Users/arduino/Documents/Arduino/libraries/Adafruit_SSD1306/Adafruit_SSD1306.cpp:24:
/Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/tools/arm/arm-none-eabi/include/c++/5.4.1/bits/exception_ptr.h:160:5: note:   'std::__exception_ptr::swap'
     swap(exception_ptr& __lhs, exception_ptr& __rhs)
     ^
Multiple libraries were found for "SD.h"
 Used: /Users/arduino/Desktop/Arduino.app/Contents/Java/hardware/teensy/avr/libraries/SD
 Not used: /Users/arduino/Desktop/Arduino.app/Contents/Java/libraries/SD
Error compiling for board Teensy 3.6.

This report would have more information with
"Show verbose output during compilation"
option enabled in File -> Preferences.

**Comment by mattrcampbell** [comment]
Ahh, it appears that the current version of Adafruit_GFX is different than the one I used.  I included a zip for it, give that a try.

**Comment by jpgagnon73** [comment]
awesome, I will try that tonight when I get home.  So from what I can see with the i2c, you connect them in parallel to pins 19 & 20 on the Teensy 3.6.  do I have to worry about the addresses on each display?

**Comment by mattrcampbell** [comment]
Yes, you will need to move the little jumper on two of the four displays for this to work.
They are NOT in parallel.  One pair goes on 18,19 and one pair goes on 38,37.
